src/aas-api-webapp-full/Controllers/AssetAdministrationShellRepositoryInterfaceApi.cs

```diff
@@ -17,9 +17,11 @@
 using AAS.API.Attributes;
 
 using Microsoft.AspNetCore.Authorization;
-using AAS.API.Models;
 using Microsoft.Extensions.Configuration;
 using AAS.API.Repository;
+using AAS_Services_Support.ADT_Support;
+using AasCore.Aas3_0_RC02;
+using AutoMapper;
```

Comment:
Please talk to me before adding additional external dependencies! Did you check the license compatibility?

Reply:
like discussed, the license is compatibel